client/client.ml

```diff
@@ -177,13 +177,13 @@ let add_user_cmd ~confdir ~conffile =
 let init ~confdir ~conffile = function
   | Some local_workdir ->
       let cwd = Sys.getcwd () in
-      let local_workdir = Server_workdirs.create ~cwd ~workdir:local_workdir in
-      let server_conf = Server_configfile.from_workdir local_workdir in
-      let profilename = Server_configfile.name server_conf in
+      let local_workdir = Server_lib.Workdirs.create ~cwd ~workdir:local_workdir in
```

Comment:
Maybe opening `Server_lib` here? Or is it for readability?

Reply:
I'd rather not. I could rename `server_lib` to server though?